data2rdf/pipelines/annotation_pipeline.py

```diff
@@ -0,0 +1,194 @@
+"""Main data2rdf Annotation pipeline"""
+
+import json
+from typing import TYPE_CHECKING, Any, Dict, Optional, Union
+
+from pydantic import (
+    BaseModel,
+    ConfigDict,
+    Field,
+    ValidationInfo,
+    field_validator,
+    model_validator,
+)
+from rdflib import Graph
+
+from data2rdf.config import Config
+from data2rdf.parsers import Parser
+from data2rdf.utils import make_prefix
+
+if TYPE_CHECKING:
+    from typing import List
+
+    from data2rdf import BasicConceptMapping
+
+
+class AnnotationPipeline(BaseModel):
```

Comment:
I remembered there was the idea to rename this class to `Data2Rdf`. What was the conclusion there? I think it would be a more intuitive name.